src/Fixer/Phpdoc/GeneralPhpdocAnnotationRemoveFixer.php

```diff
@@ -63,6 +62,10 @@ public function getPriority()
      */
     public function fix(\SplFileInfo $file, Tokens $tokens)
     {
+        if (!count($this->configuration)) {
```

Comment:
I don't like the option to use a fixer without it doing anything because the config makes no sense, it will list in the results that the fixer didn't change anything and not as a `non-useful-configured-fixer`.